README.md

```diff
@@ -27,10 +27,10 @@ npm install -g @asyncapi/generator
 
 ## Development
 
-1. Make sure you have the latest generator installed `npm install -g @asyncapi/generator`.
-1. Modify the template or it's helper functions. Adjust `test/spec/asyncapi.yml` to have more features if needed.
-1. Generate output with watcher enables `npm run develop`.
-1. Open HTML in your browser `open ./test/output/index.html`.
+1. Make sure you have the latest generator installed: `npm install -g @asyncapi/generator`.
+1. Modify the template or its helper functions. Adjust [`dummy.yml`](https://raw.githubusercontent.com/asyncapi/generator/master/test/docs/dummy.yml) to add more features if needed.
```

Comment:
please change the link here and in the other file to `https://raw.githubusercontent.com/asyncapi/generator/v1.0.0-rc.12/test/docs/dummy.yml` as we finally have it under a stable tag. Pointing directly to master is risky as it can change in master to a form that can cause errors in the template

Reply:
Agree - done :)